model/config.go

```diff
@@ -481,6 +507,71 @@ func (store *istioConfigStore) Policy(instances []*ServiceInstance, destination
 	return out
 }
 
+// DefaultProxyConfig for individual proxies
+func DefaultProxyConfig() proxyconfig.ProxyConfig {
+	return proxyconfig.ProxyConfig{
+		ConfigPath:             "/etc/istio/proxy",
+		BinaryPath:             "/usr/local/bin/envoy",
+		ServiceCluster:         "istio-proxy",
+		DrainDuration:          ptypes.DurationProto(2 * time.Second),
+		ParentShutdownDuration: ptypes.DurationProto(3 * time.Second),
+		DiscoveryAddress:       "istio-pilot:8080",
+		DiscoveryRefreshDelay:  ptypes.DurationProto(1 * time.Second),
+		ZipkinAddress:          "",
+		ConnectTimeout:         ptypes.DurationProto(1 * time.Second),
+		StatsdUdpAddress:       "",
+		ProxyAdminPort:         15000,
+	}
+}
+
+// DefaultMeshConfig configuration
+func DefaultMeshConfig() proxyconfig.MeshConfig {
+	config := DefaultProxyConfig()
+	return proxyconfig.MeshConfig{
+		EgressProxyAddress:    "istio-egress:80",
+		MixerAddress:          "",
```

Comment:
I don't understand. Why unset these? Is it for tests? If not, this needs to be istio-mixer.istio-system

Reply:
This is the same as before. I carried over defaults since istio/istio tests rely on some, and it's difficult to coordinate across two repos.